scheduler/clients/javascript/tests/requirements/requirements.spec.ts

```diff
@@ -68,7 +68,8 @@ describe('Requirements', () => {
       let user1Calendar2: Calendar | undefined
 
       beforeAll(async () => {
-        const res = await client?.user.create()
+        const userUuid = v4()
+        const res = await client?.user.create({ userId: userUuid })
```

Comment:
Ah, shouldn't be there, but it's not an issue.
It works both with providing or without providing the `userUuid`